common/include/pcl/impl/point_types.hpp

```diff
@@ -822,22 +841,28 @@ namespace pcl
     PCL_MAKE_ALIGNED_OPERATOR_NEW
   };
 
-
+  struct _PointXY
```

Comment:
Is this new stuct necessary? Couldn't the new stuff just be added to the existing PointXY struct?

Reply:
I have removed this.